### DISCUSSION TITLE
Milestone이랑 PR, issue 정리했습니다

### DISCUSSION THREAD

**Discussion by Jun0zo** [osamhack2022-v2/WEB_CLOUD_OmilZomil_NAVYeffect#157] [Show and tell]
제목 그대로 저 3개 정리 했습니다.

PR이랑 issue 정리해서 milestone에 넣었고 
PR이랑 issue는 모두 label만들어서 분류했습니다.
PR 제목중에서 막 PR test 이런거 많았는데 다 바꾸느라 힘들었습니다 ㅠ
PR 제목으로 어떤 내용이 수정되었는지 알 수 없으면 commit 제목으로 유추해야 하기 때문에
PR제목과 내용, 그리고 commit 제목이 중요한 것 같습니다.
이번 기회에 commit제목과 PR제목의 중요성을 다시 깨닫습니다...
이렇게 정리된 PR로 이제 release를 준비해야하는데 일단 한번 만들어보았으니 참고하시길

각 position별로 구분해보았습니다.
=========================================================
#2 정복인식모델 구현 @Jun0zo
#24 샘브레이 동적복 인식모델 수정 @Jun0zo
#72 Uniform 인식모델 이미지 저장기능 추가 @Jun0zo
#76 Uniform 샘브레이 이미지 저장기능 추가 @Jun0zo
#119 Demo video 전용 모듈 구현 @Jun0zo

#87 Front&Back 도커화작업  @greenrain78

#122 부대출입기록 및 장병 데이터베이스 스키마 및 ORM설계 @dos-m0nk3y
#128 fastapi 공식 repo에 기반한 디렉토리 구조 개편 @dos-m0nk3y
#131 계급, 부대 DB 구현 완료 @dos-m0nk3y
#138 유저 테이블, ORM 및 엔드포인트 설계 완료 @dos-m0nk3y

=========================================================

오늘 알게 된 사실
1. commit에 #PR 번호 붙이면 해당 commit에 PR이 reference되고 그 reference된 PR에도 해당 commit이 나온다 ㄷㄷ (commit뿐만 아니라 github 내에서 글을 쓸 수 있는 환경에서는 다 가능한듯)
2. PR제목에 #PR 하면 reference가 안되는 것 같다. (이건 더 알아봐야할 듯)

**Comment by dos-m0nk3y** [comment]
#138 유저 테이블, ORM 및 엔드포인트 설계 완료 @dos-m0nk3y
마지막에 추가 부탁드립니다.

**Comment by dos-m0nk3y** [comment]
그리고 표현은 통일성 있게 다듬어서 release 하는게 좋을거 같습니다.

**Reply by Jun0zo** [reply]
좋습니다! 저게 최종본은 아니라서 다른사람들도 추가요청사항 있으면 말씀해주십쇼
